### PR TITLE
added jest test and moved to ES6

### PR DESCRIPTION
**Pull request by saikrishna321**

### CODE REVIEW

**Comment by BugDiver** [comment]
@saikrishna321 Thanks for the effort. We have also put some typescript code as well for the same thing. You might wanna check that out. It can be found under `typescript` branch.

**Comment by saikrishna321** [comment]
Thanks for pointing out!! Was not aware of that. Will take a look.

Does Taiko support TS? Any reason why typescript hasn't been merged to master ?

**Comment by BugDiver** [comment]
> Does Taiko support TS?

Well, It's written in `commonjs` and does not have typings (`.d.ts`). But I think It should be working with typescript. Most `commonjs` modules work with typescript.

> Any reason why typescript hasn't been merged to master
No specific reason. I ported the code out of curiosity. If you think it makes sense then we can merge that.

FWIW I would like to point out https://github.com/getgauge/taiko/issues/439 as well.
Please have a look and add your thoughts.

**Comment by saikrishna321** [comment]
@BugDiver I'm happy to just stay with commonjs modules, as this is gonna be a lean module. 

If you are happy with PR we can merge to master

File: package.json [hunk 1]
```diff
@@ -2,13 +2,23 @@
...
   "license": "MIT",
   "dependencies": {
-    "speedline": "^1.4.2"
+    "@babel/core": "^7.3.4",
+    "babel-jest": "^24.3.1",
+    "jest": "^24.3.1",
+    "speedline": "^1.4.2",
+    "@babel/cli": "^7.2.3",
+    "@babel/preset-env": "^7.3.4",
+    "taiko": "git://github.com/getgauge/taiko.git"
```
Comment (BugDiver) [inline package.json]: @saikrishna321 Shouldn't all these dependencies are required only for development?
We only need `speedline` for as production dependency.
Could you please move them to `dev-dependencies`?

**Pull request merged**